unitTests/ui/FieldValueTest.ts

```diff
@@ -358,37 +356,30 @@ export function FieldValueTest() {
       const dateTimeString = DateUtils.dateTimeToString(new Date(parseInt(fakeResult.raw.date)), fullDateOptions);
 
       initializeFieldValueComponent(options, fakeResult);
-      expect(
-        $$(test.cmp.element)
-          .find('span')
-          .getAttribute('title')
-      ).toEqual(dateString);
+      expect($$(test.cmp.element).find('span').getAttribute('title')).toEqual(dateString);
 
       options.helper = 'dateTime';
       initializeFieldValueComponent(options, fakeResult);
-      expect(
-        $$(test.cmp.element)
-          .find('span')
-          .getAttribute('title')
-      ).toEqual(dateTimeString);
+      expect($$(test.cmp.element).find('span').getAttribute('title')).toEqual(dateTimeString);
 
       options.helper = 'emailDateTime';
       initializeFieldValueComponent(options, fakeResult);
-      expect(
-        $$(test.cmp.element)
-          .find('span')
-          .getAttribute('title')
-      ).toEqual(dateTimeString);
+      expect($$(test.cmp.element).find('span').getAttribute('title')).toEqual(dateTimeString);
     });
 
     it("should not show a full date tooltip if it doesn't have the helper is not a date", () => {
       initializeFieldValueComponent({ field: '@string' });
 
-      expect(
-        $$(test.cmp.element)
-          .find('span')
-          .hasAttribute('title')
-      ).toBe(false);
+      expect($$(test.cmp.element).find('span').hasAttribute('title')).toBe(false);
+    });
+
+    it('should display a value if it is a falsy value', () => {
+      [0, -0, 'false', 'null', 'undefined'].forEach(falsy => {
```

Comment:
Should the test take `false`, `null` and `undefined` as primitives instead of strings?

Reply:
I can add false as primitive, but for `null` or `undefined`, no, because for these we actually want to not display anything. That's what the `isNullOrUndefined` is for.